frontend/src/components/WorkspaceUtilityRail.css

```
@@ -0,0 +1,210 @@
+}
+
+.utility-icon-btn.active {
+  color: var(--accent, #d97757);
+  box-shadow: inset 2px 0 0 var(--accent, #d97757);
+}
+
+.utility-icon-btn svg {
+  width: 18px;
+  height: 18px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)